static/app/views/issueDetails/streamline/sidebar/activitySection.tsx

```diff
@@ -82,7 +89,19 @@ function TimelineItem({
         )
       }
     >
-      {typeof message === 'string' ? (
+      {item.type === GroupActivityType.NOTE && editing ? (
+        <NoteInputWithStorage
+          itemKey={item.id}
+          storageKey={`groupinput:${item.id}`}
+          source="issue-details"
+          text={item.data.text}
+          noteId={item.id}
+          onUpdate={n => {
+            handleUpdate(item, n);
+            setEditing(false);
+          }}
+        />
```

Comment:
Could we have the edit button have a Save/Cancel button? Currently there's no way back from editing without a page refresh.

Reply:
yup - will add a cancel button!